scripts/Synapse/extractTSV.ipynb

```diff
@@ -0,0 +1,336 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": 21,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [
+        {
+          "data": {
+            "application/vnd.livy.statement-meta+json": {
+              "execution_finish_time": "2023-05-10T20:44:54.05801Z",
+              "execution_start_time": "2023-05-10T20:44:51.1435512Z",
+              "livy_statement_state": "available",
+              "parent_msg_id": "59c23745-07c1-4254-a899-54a045cd8f1d",
+              "queued_time": "2023-05-10T20:44:50.9442763Z",
+              "session_id": "24",
+              "session_start_time": null,
+              "spark_jobs": null,
+              "spark_pool": "marcellepool",
```

Comment:
Maybe this should be renamed now? ;)

Reply:
We are going to need to add some interpolation here so that when we deploy to different environments we set the pool correctly.

Reply:
Actually now that I'm looking at this again, this output metadata from this cell running. We don't want to include any of this in the repo. I'll push a commit to clean up this notebook.